index.html

```diff
@@ -601,6 +601,180 @@ <h2 class="sg-section-title structure">Forms</h2>
                         </header>
                         <div class="sg-section-content structure">
                             
+
+    
+        <h3 class="section-sub-title">Example Form</h3>
+
+        <form action="" method="post">
+            <div class="error-block form-error">
+                <p class="error">This is an example form error</p>
+            </div>
+
+            
+                <fieldset class="text-fieldset">
+                    <div class="control-group text required">
+                        <label for="field-name" class="control-label">
+                            Text input
+                            <span class="asterisk">*</span>
+                        </label>
+                        <div class="controls">
+                            <input class="text-input" id="field-name" type="text" name="name" placeholder="Text input" aria-required="true">
+                        </div>
+                    </div>
+
+                    <div class="control-group email required">
+                        <label for="field-email" class="control-label">
+                            Email input
+                            <span class="asterisk">*</span>
+                        </label>
+                        <div class="controls">
+                            <input class="email-input" id="field-email" type="email" name="email" placeholder="Email input" aria-required="true">
```

Comment:
Hmm actually no. According to MDN " The first such element in the document with an ID matching the value of the for attribute is the labeled control for this label element."
https://developer.mozilla.org/en/docs/Web/HTML/Element/label

FAIL